ironfish/src/metrics/metricsMonitor.ts

```diff
@@ -80,6 +83,15 @@ export class MetricsMonitor {
     return meter
   }
 
+  recordP2PMessage(peerId: Identity): void {
```

Comment:
This function is gonna cause a big performance and memory leak by adding a new meter for every connected peer because these are not cleaned up and stopped.

Reply:
Good catch, I'll clean them up

Reply:
Since we add them by using `addMeter` function I think these all will be cleaned up actually. The `Map<Identity, Meter>` will not be deleted or anything but that should be taken care of by garbage collection? Is there something else you think we should do in cleanup